src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -579,38 +572,37 @@ like `r'About $applicationName'`,
 because sometimes the strings contain variables with a `$` prefix.
 The variables are expanded by parameterized localization methods:
 
-<!-- skip -->
+<?code-excerpt "add_language/lib/nn_intl.dart (Raw)"?>
 ```dart
 @override
-String get aboutListTileTitleRaw => r'About $applicationName';
+String get pageRowsInfoTitleRaw => r'$firstRow–$lastRow of $rowCount';
 
 @override
-String aboutListTileTitle(String applicationName) {
-  final String text = aboutListTileTitleRaw;
-  return text.replaceFirst(r'$applicationName', applicationName);
-}
+String get pageRowsInfoTitleApproximateRaw =>
+    r'$firstRow–$lastRow of about $rowCount';
 ```
 
 The date patterns and symbols of the locale will also need to
 be specified. In the source code, the date patterns and symbols
 are defined like this:
 
-<!-- skip -->
+<?code-excerpt "add_language/lib/nn_intl.dart (Date)"?>
 ```dart
 const nnLocaleDatePatterns = {
   'd': 'd.',
   'E': 'ccc',
-  //...
-}
+  'EEEE': 'cccc',
+  'LLL': 'LLL',
+```
 
+<?code-excerpt "add_language/lib/nn_intl.dart (Date2)"?>
+```dart
 const nnDateSymbols = {
   'NAME': 'nn',
   'ERAS': <dynamic>[
     'f.Kr.',
     'e.Kr.',
   ],
-  // ...
-}
```

Comment:
If someone copy/pastes this, it'll result in an error since it's an incomplete map. Is there a way we can preserve the correctness of the objects here and elsewhere? I feel like this will likely be a source of friction for our users.